docs/howto/advanced/shutdown.md

```diff
@@ -6,13 +6,18 @@ A worker will keep running until:
 - [task.cancel](https://docs.python.org/3/library/asyncio-task.html#asyncio.Task.cancel) is called on the task created from `app.run_worker_async`
 
 When a worker is requested to stop, it will attempt to gracefully shut down by waiting for all running jobs to complete.
-If a `shutdown_timeout` option is specified, the worker will attempt to abort all jobs that have not completed by that time. Cancelling the `run_worker_async` task a second time also results in the worker aborting running jobs.
+If a `shutdown_graceful_timeout` option is specified, the worker will attempt to abort all jobs that have not completed by that time. Cancelling the `run_worker_async` task a second time also results in the worker aborting running jobs.
+
+The worker will then wait for all jobs to complete.
+
 
 :::{note}
-The worker aborts its remaining jobs by calling [task.cancel](https://docs.python.org/3/library/asyncio-task.html#asyncio.Task.cancel) on the underlying asyncio task that runs the job.
+The worker aborts its remaining jobs by:
+- setting `JobContext.should_abort` to `True`
+- calling [task.cancel](https://docs.python.org/3/library/asyncio-task.html#asyncio.Task.cancel) on the underlying asyncio task that runs the job when the job is asynchronous.
+Jobs that do not respect the request to abort will prevent the worker from shutting down.
+For more information, see {doc}`howto/cancellation`.
```

Comment:
It's `howto/advanced/cancellation`. Then, building the docs should run through without a warning, and all checks pass.